MetadataExtractor/DirectoryExtensions.cs

```diff
@@ -348,6 +348,10 @@ public static byte[] GetByteArray(this Directory directory, int tagType)
             if (o == null)
                 return null;
 
+            var strvalue = o as StringValue;
+            if (strvalue != null)
+                return strvalue.Bytes;
+
             byte[] bytes;
 
             var rationals = o as Rational[];
```

Comment:
Shouldn't a few other methods in `DirectoryExtensions` become `StringValue`-aware? At the very least, `GetString` should, with an optional encoding argument.

Maybe also `DirectoryExtensions.GetStringValue`, though I don't think that should do any coercion of types as going from, say, a double, to a string, to a byte[] seems pretty dumb and error prone.